app/server/api/transfer_usage_api.py

```diff
@@ -1,7 +1,6 @@
 from flask import Blueprint, request, make_response, jsonify
 from flask.views import MethodView
 from sqlalchemy import desc
```

Comment:
`from sqlalchemy import desc` is also unused it seems?

Reply:
Hm, vulture didn't pick those up for some reason. Will delete when I log on in the AM!

Reply:
no worries!